Marlin/src/lcd/extui/lib/dgus/DGUSScreenHandler.cpp

```diff
@@ -395,18 +395,21 @@ void DGUSScreenHandler::HandleTemperatureChanged(DGUS_VP_Variable &var, void *va
     default: return;
     #if HOTENDS >= 1
       case VP_T_E0_Set:
+        if(newvalue > HEATER_0_MAXTEMP)  newvalue = HEATER_0_MAXTEMP;
         thermalManager.setTargetHotend(newvalue, 0);
         acceptedvalue = thermalManager.temp_hotend[0].target;
         break;
     #endif
     #if HOTENDS >= 2
       case VP_T_E1_Set:
+        if(newvalue > HEATER_0_MAXTEMP)  newvalue = HEATER_1_MAXTEMP;
         thermalManager.setTargetHotend(newvalue, 1);
         acceptedvalue = thermalManager.temp_hotend[1].target;
         break;
     #endif
     #if HAS_HEATED_BED
       case VP_T_Bed_Set:
+        if(newvalue > BED_MAXTEMP)  newvalue = BED_MAXTEMP;
```

Comment:
This would not seem to matter since `newvalue` is not being retained, and `thermalManager` will reject the higher `newvalue` as certainly as it will reject the reduced `newvalue`.

Reply:
My other question is, why would the rejection of the set value matter to whether or not the display updates here? The only thing that should matter is whether the new value for display differs from what was previously displayed.